manifests/certonly.pp

```diff
@@ -97,7 +97,7 @@
     command     => $command,
     path        => $::path,
     environment => $execution_environment,
-    onlyif      => "test -f ${live_path} && ( openssl x509 -in ${live_path} -text -noout | grep -oE 'DNS:[^\s,]*' | sed 's/^DNS://g;'; echo '${verify_domains}' | tr ' ' '\\n') | sort | uniq -c | grep -qv '^[ \t]*2[ \t]'",
+    unless      => "/usr/local/sbin/letsencrypt-domain-validation ${live_path} ${verify_domains}",
```

Comment:
Why not a require to `File['/usr/local/sbin/letsencrypt-domain-validation']` ?

Reply:
That's a good idea! :)